### DISCUSSION TITLE
Consider Token Frequency when calculating ratio

### DISCUSSION THREAD

**Discussion by JayPalm** [rapidfuzz/RapidFuzz#117] [Ideas]
It doesn't seem like any of the scorers consider the token frequency (how many times a word/substring appears), making it difficult to match phrases with repeating terms. For my current need, I'm able to simply filter out the reoccurring word, but it would be cool to have a frequency-sensitive scorer.

**Comment by maxbachmann** [comment]
>  For my current need, I'm able to simply filter out the reoccurring word, but it would be cool to have a frequency-sensitive scorer.

Do you mean it should ignore duplicated words? Fuzz.token_set_ratio is a scorer which ignores duplicated words.
